extensions/amp-script/amp-script.md

```diff
@@ -46,61 +44,101 @@ Allows rendering of custom UI components running on third-party JavaScript.
       </div>
     </td>
   </tr>
+  <tr>
+    <td class="col-fourty"><strong>Examples</strong></td>
+    <td>
+      <ul>
+        <li><a href="https://github.com/ampproject/amphtml/tree/master/examples/amp-script">Unannotated code samples</a></li>
+      </ul>
+    </td>
+  </tr>
+  <tr>
+    <td class="col-fourty"><strong>Tutorials</strong></td>
+    <td>
+      <ul>
+        <li><a href="https://amp.dev/documentation/guides-and-tutorials/develop/custom-javascript">Getting started with amp-script</a></li>
+        <li><a href="https://amp.dev/documentation/guides-and-tutorials/develop/custom-javascript-tutorial?format=websites">Custom password requirements with amp-script</a></li>
+      </ul>
+    </td>
+  </tr>
 </table>
 
 ## Overview
 
-The `amp-script` component allows you to render widgets and other UI using custom third-party JavaScript, e.g. a React component.
+The `amp-script` component allows you run custom JavaScript to render UI e.g. React components.
```

Comment:
I'd reword as:

he `amp-script` component allows you run custom JavaScript to render UI elements, such as a React component.

Reply:
Done.